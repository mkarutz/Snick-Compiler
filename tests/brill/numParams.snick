proc main ()
    f(1,2,3);
end

proc f(val int a, val int b)
    write a + b;
end
