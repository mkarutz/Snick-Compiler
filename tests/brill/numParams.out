Failure: Number of actual parameters does not match expected number of arguments.
Compilation Failed
